cpp/oneapi/dal/data/undirected_adjacency_array_graph.cpp

```diff
@@ -35,62 +35,65 @@ template class ONEAPI_DAL_EXPORT undirected_adjacency_array_graph<empty_value,
                                                                   empty_value,
                                                                   empty_value,
                                                                   std::int32_t,
-                                                                  std::allocator<empty_value>>;
+                                                                  std::allocator<char>>;
 
-using graph32 = undirected_adjacency_array_graph<empty_value,
-                                                 empty_value,
-                                                 empty_value,
-                                                 std::int32_t,
-                                                 std::allocator<empty_value>>;
+using graph_default = undirected_adjacency_array_graph<empty_value,
```

Comment:
what will be the name for 64 bit version of the graph?

Reply:
no current solution for it. graph64 or graph_with_int64_t_index_type??? Note: these usings are only for compact instances. they are not exposed to public.

Reply:
can we discuss and agree offline on the name